### DISCUSSION TITLE
Remix v3/React Router 7 has been released, what's next for @rvf/remix?

### DISCUSSION THREAD

**Discussion by alvinvin00** [airjp73/rvf#403] [Q&A] [answered]
as per title, React Router v7 was released few hours ago, and following Remix team's plan to assimilate what would have been Remix v3 into React Router v7, what's the next plan forward for `@rvf/remix`? I tried to migrate my Remix project and i got blocked by rvf when doing `npm install`

**Comment by airjp73** [comment, marked as answer]
Hi! We'll need to make a `@rvf/react-router` package to support RRv7. It will hopefully be a pretty direct port of `@rvf/remix` with minor changes, but I haven't dug into the changelog yet to be sure.

**Reply by rcnet4** [reply]
Wow nice! Currently upgraded to RRv7, looking forward to @rvf/react-router :)

**Reply by rcnet4** [reply]
Thank you very much for @rvf/react-router
